rmf_demo_panel/README.md

```diff
@@ -0,0 +1,52 @@
+# RMF Demo Panel
+
+This will be a replacement of rviz's [rmf_panel](https://github.com/osrf/rmf_schedule_visualizer)
```

Comment:
The [RMF Panel](https://github.com/osrf/rmf_demos/tree/develop/dispatcher_demo/rmf_rviz_plugin) is located in `rmf_demos` repo and not `rmf_schedule_visualizer`. I'd recommend just deleting this line since we have plans to delete the old package anyways.